i18n-util/src/UrlUtil.js

```diff
@@ -1,4 +1,12 @@
 
+
+const getUrlParameter = (name) => {
+  name = name.replace(/[\\[]/, '\\[').replace(/[\]]/, '\\]');
```

Comment:
style-nit: We shouldn't need to reassign a given value here - instead, let's use another variable.